### PR TITLE
sybaseASE: new resource agent

### PR DESCRIPTION
**Pull request by oalbrigt**
Based on https://github.com/ClusterLabs/resource-agents/pull/398 with fixes and improvements.

### CODE REVIEW

File: heartbeat/sybaseASE [hunk 1]
```diff
@@ -0,0 +1,895 @@
...
+#                <!--Checks to see if we can read from the mountpoint -->
+#                <action name="status" depth="10" timeout="100" interval="120" />
+#                <action name="monitor" depth="10" timeout="100" interval="120" />
+#
+#                <action name="meta-data" timeout="5" />
+#                <action name="validate-all" timeout="5" />
+#     The timeout value is not supported by Redhat in RHCS5.0.
+#
+# (3) This script should be put under /usr/share/cluster. Its owner should be "root" with
+#     execution permission.
```
Comment (krig) [inline heartbeat/sybaseASE]: Is this really true, still?

File: heartbeat/sybaseASE [hunk 2]
```diff
@@ -0,0 +1,895 @@
...
+# Declare some variables we will use in the script. Please don't change their values. #
+#######################################################################################
+declare login_string=""
+declare RUNSERVER_SCRIPT=$OCF_RESKEY_sybase_home/$OCF_RESKEY_sybase_ase/install/RUN_$OCF_RESKEY_server_name
+declare CONSOLE_LOG=$OCF_RESKEY_sybase_home/$OCF_RESKEY_sybase_ase/install/$OCF_RESKEY_server_name.log
+
+##################################################################################################
+# This function will be called by rgmanager to get the meta data of resource agent "sybaseASE". #
+# NEVER CHANGE ANYTHING IN THIS FUNCTION.
+##################################################################################################
```
Comment (krig) [inline heartbeat/sybaseASE]: How about dropping the "never change anything..." comments? :)

File: heartbeat/sybaseASE (hunk 1 above)
Comment (oalbrigt) [reply heartbeat/sybaseASE]: Oh. It was made for RGManager first, and then ported, so I'll remove that part :)

File: heartbeat/sybaseASE (hunk 2 above)
Comment (oalbrigt) [reply heartbeat/sybaseASE]: Good idea. Wont be able to fix bugs if we cant change anything.

**Pull request merged**